### PR TITLE
Dd 1335

### PR DESCRIPTION
**Pull request by ericdevries**
Fixes DD-1335

# Description of changes

Cleanup of notifications after a certain amount of days. 

What this also shows is my idea for wrapping the postgres stuff inside some class that just lets us use it without dealing with the terrible psycopg API. I believe this is the right level of abstraction here, in the context of this project. 

If we were to write unit tests for this, it would be tricky because of the direct usage of the psycopg objects in the logic, but abstracting that away will probably make the code too complex, just for the sake of "clean code". 

If you disagree with my opinion, do not hesitate to let me know :).

Also on code formatting, I formatted this with "black" (see https://pypi.org/project/black/) to get rid of custom formatting. I find it very pleasing to look at as every python code will look identical. 

# How to test

# Related PRs

(Add links)

*

# Notify

@DANS-KNAW/dataversedans

### CODE REVIEW

**Comment by PaulBoon** [comment]
About the formatting (black); When using PyCharm I don't remember feeling the need to reformat. Could this 'black' be integrated in PyCharm or in the build process somehow not having me to think about it?

**Review by janvanmansum — CHANGES_REQUESTED** [review]
@ericdevries : I have restructured the project. Please refactor this PR to conform to the new style. 

See also: https://dans-knaw.github.io/dans-datastation-tools/dev/

If it is more convenient you might also do  new PR.

**Comment by ericdevries** [comment]
> About the formatting (black); When using PyCharm I don't remember feeling the need to reformat. Could this 'black' be integrated in PyCharm or in the build process somehow not having me to think about it?

I don't use pycharm, but I know it is properly supported in Vim and visual studio code. It looks a bit more cumbersome for IntelliJ products though.

**Comment by ericdevries** [comment]
Also seeing as the development guidelines state that the PyCharm formatter should be used, perhaps we should not go down the black route anymore. 

My only problem is that I don't use pycharm

File: src/datastation/dv_notifications_cleanup.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+    notifications = Notifications(
+        dataverse_client, 
+        dry_run=args.dry_run
+    )
+    
+    notifications.cleanup(args.days_old)
+
+
+def positive_int(value):
```
Comment (janvanmansum) [inline src/datastation/dv_notifications_cleanup.py:34]: Move to `utils` package

File: src/datastation/dataverse/notifications.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
+from datastation.common.csv import CsvInput
+from datastation.common.database import Database
+from datastation.dataverse.builtin_users import User
+from datastation.dataverse.dataverse_client import DataverseClient
+import logging
+from datetime import timedelta
+
+
+class Notifications:
+    """Imports users from a CSV file into a Dataverse instance. The CSV file can be exported from EASY or manually
```
Comment (janvanmansum) [inline src/datastation/dataverse/notifications.py:10]: Remove the doc comment (copy-paste error?)

**Review by janvanmansum — APPROVED** [review]

**Pull request merged**